Week1/Solution/1. Prime or Not.

```diff
@@ -0,0 +1,12 @@
+#write a python script to Enter any Number and check it is prime or not.
+n=int(input('Enter Any Number:: '))
+c=0 #it called count.it is used to define how many times the number is divided.
+i=1 
+for i in range(i,n+1):
```

Comment:
EXPLAIN FOR LOOP.

Reply:
Prime number is a number which is divided by 1 and itself.
In program for loop,is used to check the given  number is divided by itself and 1.
Like ex. If the user input 7 then the loop will be check number is divided two time.if it is true than it is a prime number otherwise it is not a prime number.